rest/src/main/java/org/jboss/pnc/rest/endpoint/BuildConfigurationSetEndpoint.java

```diff
@@ -186,8 +186,7 @@ public Response build(
     public Response addConfiguration(
             @ApiParam(value = "Build Configuration Set id", required = true) @PathParam("id") Integer id,
             BuildConfigurationRest buildConfig) {
-        buildConfigurationSetProvider.addConfiguration(id, buildConfig.getId());
-        return Response.ok().build();
+        return buildConfigurationSetProvider.addConfiguration(id, buildConfig.getId());
```

Comment:
I actually need to return a different Status code if that BuildConfig alread exists in the set. An alternative could be throwing a runtime exception that would be caught in Endpoint